resources/group.py

```diff
@@ -26,7 +26,7 @@ def post(self):
             conn = mysql.connect()
             cursor = conn.cursor()
 
-            if permission != 'pf':
+            if permission != 'pf' or permission != 'su':
```

Comment:
Why would students be able to create classes? This sounds like something only a professor should be able to do

Reply:
ST stands for Student, PF stands for Professor and SU stands for Super User

Reply:
I see